marlowe-runtime/doc/resources.md

```diff
@@ -0,0 +1,55 @@
+# Recommended Resources for Deploying Marlowe Runtime
+
+This document summarizes recommendations for the computational resources required to deploy Marlowe Runtime on public networks (`mainnet`, `preprod`, and `preview`).
+
```

Comment:
I suggest we add this as a cross-reference: 

For more information about how to get started using the Cardano testnets, please see [Getting started](https://docs.cardano.org/cardano-testnet/getting-started).

Reply:
Fixed in 12ffeb4d965e98f5ea3afc0a1ec54c7d3ce5e840.